dev_tools/profiling/benchmark_simulators.py

```diff
@@ -26,17 +26,20 @@
 _DENSITY = 'density_matrix'
 
 
-test_device = cirq.google.XmonDevice(
-    measurement_duration=cirq.Duration(nanos=1000),
-    exp_w_duration=cirq.Duration(nanos=20),
-    exp_11_duration=cirq.Duration(nanos=50),
-    qubits=[cirq.GridQubit(0, k) for k in range(100)],
-)
+def test_device():
```

Comment:
I'd be surprised if the specific device matters at all to this benchmark. In my experience, simulators don't really care what the device is outside of some very basic validation.

Reply:
You are right. A little digging shows that original bencmhark did not have the device - the performance was benchmarked purely based on the circuits. The device was introduced for XmonSimulator specifically when it did some extra validation (https://github.com/quantumlib/Cirq/pull/1405). Now, that XmonSimulator is removed this can be removed too freeing us from cirq.google dependencies! Yay.